projects/packages/videopress/src/client/state/resolvers.js

```diff
@@ -420,6 +421,27 @@ const getPlaybackToken = {
 	},
 };
 
+const getVideoPressSettings = {
+	isFulfilled: state => {
+		return state?.siteSettings !== undefined;
+	},
+	fulfill: () => async ( { dispatch } ) => {
+		try {
+			const { videopress_videos_private_for_site: videoPressSitePrivacySetting } = await apiFetch( {
```

Comment:
I know the PR is in progress, but should we rename the prop to `videoPressVideosPrivateForSite`?

Reply:
Makes sense, so we keep it consistent on client, backend and JSON body. I will update it.

Reply:
Updated the code, as well as the PR description. Thanks for the suggestion!